code/datums/outfits/ert/ap_eridani.dm

```diff
@@ -0,0 +1,164 @@
+/datum/outfit/admin/ert/ap_eridani
+	name = "Eridani Asset Protection Specialist"
+
+	uniform = /obj/item/clothing/under/rank/security/eridani
+	accessory = /obj/item/clothing/accessory/holster/thigh
+	accessory_contents = list(/obj/item/gun/projectile/automatic/x9 = 1)
+	suit = /obj/item/clothing/suit/space/void/cruiser
+	head = /obj/item/clothing/head/helmet/space/void/cruiser
+	mask = /obj/item/clothing/mask/gas/tactical
+	shoes = /obj/item/clothing/shoes/swat/ert
+	gloves = /obj/item/clothing/gloves/combat
+	glasses =  /obj/item/clothing/glasses/sunglasses
+	suit_store = /obj/item/gun/energy/rifle
+	back = /obj/item/storage/backpack/satchel_sec
+	belt = /obj/item/storage/belt/military
+	id = /obj/item/card/id/distress/ap_eridani 
+	l_ear = /obj/item/device/radio/headset/distress
+	l_pocket = /obj/item/grenade/chem_grenade/teargas
+	r_pocket = /obj/item/tank/emergency_oxygen/double
+
+	backpack_contents = list(
+		/obj/item/storage/box/survival = 1,
+		/obj/item/reagent_containers/hypospray/autoinjector/survival = 1,
+		/obj/item/storage/firstaid/regular = 1,
+		/obj/item/clothing/head/softcap/eri = 1,
+		/obj/item/gun/energy/gun/nuclear = 1,
+		/obj/item/storage/belt/utility/very_full = 1
+	)
+
+	belt_contents = list(
+		/obj/item/ammo_magazine/c45x = 3,
+		/obj/item/device/flash = 1,
+		/obj/item/handcuffs/ziptie = 1,
+		/obj/item/melee/baton/stunrod = 1,
+		/obj/item/device/flashlight/maglight = 1,
+		/obj/item/shield/riot/tact = 1,
+		/obj/item/grenade/flashbang = 1
+	)
+
+/datum/outfit/admin/ert/ap_eridani/get_id_access()
+	return get_distress_access()
```

Comment:
Here you specify the desires access as get_distress_access()
(Which is: `access_legion, access_distress, access_maint_tunnels, access_external_airlocks, access_security, access_engine, access_engine_equip, access_medical, access_research, access_atmospherics, access_medical_equip, access_construction`)